tools/c64/c64upload.py

```
@@ -0,0 +1,59 @@
+
+    with open(source_path, "rb") as f:
+        ftp.storbinary(f"STOR {target_path}", f)
+
+    ftp.dir("SD")
+    ftp.quit()
+    print("Deployment complete.")
+
+    if args.mount:
+        print("Mounting program on C64...")
+
+        requests.put(
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)